golang/pkg/cli/config_file.go

```diff
@@ -301,6 +301,14 @@ func SaveSettings(settings *Settings) {
 
 		// If settingspath is default, we create the directory for it
 		if settings.SettingsFilePath == settingspath {
+			if _, err := os.Stat(userconfdir); errors.Is(err, os.ErrNotExist) {
```

Comment:
why is useconfdir variable not camelCased? :dagger:

Reply:
changed in every instance